pom.xml

```diff
@@ -4,147 +4,63 @@
-        <junit.version>4.12</junit.version>
-        <immutables.version>2.5.5</immutables.version>
-        <mockito.version>2.23.4</mockito.version>
-        <assertj-core.version>3.8.0</assertj-core.version>
-        <slf4j-api.version>1.7.25</slf4j-api.version>
-        <picocli.version>4.2.0</picocli.version>
-        <slf4j-log4j12.version>1.7.28</slf4j-log4j12.version>
-        <google-cloud-bom.version>0.145.0</google-cloud-bom.version>
-        <java-client.version>1.26.0</java-client.version>
-        <pipeline5.version>5.18.1943</pipeline5.version>
-        <pipeline-events.version>3.0.0</pipeline-events.version>
-    </properties>
-
     <repositories>
         <repository>
-            <id>hmf-maven-repository-release</id>
-            <url>gs://hmf-maven-repository/release</url>
+            <id>artifact-registry</id>
+            <url>artifactregistry://europe-west4-maven.pkg.dev/hmf-build/hmf-maven</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
         </repository>
     </repositories>
 
+    <properties>
+        <main.class>com.hartwig.snpcheck.SnpCheckMain</main.class>
+    </properties>
+
     <dependencies>
 
         <dependency>
             <groupId>com.hartwig.api</groupId>
             <artifactId>java-client</artifactId>
-            <version>${java-client.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>com.hartwig</groupId>
-            <artifactId>pipeline-events</artifactId>
-            <version>${pipeline-events.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>com.hartwig</groupId>
-            <artifactId>cluster</artifactId>
-            <version>${pipeline5.version}</version>
-        </dependency>
-
-        <dependency>
-            <groupId>org.immutables</groupId>
-            <artifactId>value</artifactId>
-            <version>${immutables.version}</version>
+            <version>1.30.1</version>
         </dependency>
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>${slf4j-api.version}</version>
+            <groupId>com.hartwig.events</groupId>
+            <artifactId>events-pipeline</artifactId>
         </dependency>
         <dependency>
-            <groupId>info.picocli</groupId>
-            <artifactId>picocli</artifactId>
-            <version>${picocli.version}</version>
+            <groupId>com.hartwig.events</groupId>
+            <artifactId>events-pubsub</artifactId>
         </dependency>
-
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-            <version>${slf4j-log4j12.version}</version>
-            <scope>runtime</scope>
+            <groupId>com.hartwig.cloud-sdk</groupId>
+            <artifactId>cli</artifactId>
```

Comment:
I could be mistaken, but I think you need a `<version>2.5.0</version>` here. Otherwise you will default to the project version, which in this case is `local-SNAPSHOT`.